app/councils.json

```diff
@@ -121,6 +122,7 @@
     "parkingBoundary": "Ashbrooke and Thornholme",
     "boundaryLink": "http://www.sunderland.gov.uk/CHttpHandler.ashx?id=18284&p=0&fsize=883kb&ftype=Ashbrooke%20and%20Thornholme%20CPMS.PDF",
     "boundaryPage": "parking-boundary-map",
+    "hmo": "phone",
```

Comment:
@lizziebruce What does this do? Doesn't look like you've used it in the HTML to me